### ISSUE TITLE
UI Bug when displaying one-element dropdown

### ISSUE THREAD

**Issue by Lyczeq** [SAP/luigi#2408]
**Description**

When using `appSwitcher` and the `items` attribute in the returned object includes only one element there is a UI bug related to Luigi's navigation.  It tries to display a one-element list, so there is just an arrow with a dot. 
It appears on every browser.

![image](https://user-images.githubusercontent.com/56163767/144429483-e78c3d2f-a28e-48e1-b3fe-845aa603533c.png)

**Expected result**
When the `items` array includes only one element there shouldn't be an option to dropdown 

**Actual result**

The one-element list is displayed with a UI bug that shows an arrow and a dot.

**Steps to reproduce**

1. Use the code below or create similar `appSwitcher` in the configuration that returns one element array with a proper link.
```js
appSwitcher: {
  items: [{
    title: 'Cluster Overview',
    link: '/clusters',
  }]
}
```
2. Click on the Dropdown you've created.

**Comment by UlianaMunich** [comment]
Hi @Lyczeq, thanks for the report. Will take care of it. Stay tuned

**Comment by UlianaMunich** [comment]
Dear @Lyczeq , the fix will be released with the upcoming Luigi version.

**Comment by UlianaMunich** [comment]
Hi @Lyczeq, a fix has been released under Luigi v1.20.1.
